scipy/stats/_resampling.py

```diff
@@ -1224,6 +1224,21 @@ def permutation_test(data, statistic, *, permutation_type='independent',
     only one sample, then the null distribution is formed by independently
     changing the *sign* of each observation.
 
+    .. warning::
+        The p-value is calculated by counting the elements of the null
+        distribution that are as extreme or more extreme than the observed
+        value of the statistic. Due to the use of finite precision arithmetic,
+        some statistic functions return numerically distinct values when the
+        theoretical values would be exactly equal. In some cases, this could
+        lead to a large error in the calculated p-value. `permutation_test`
+        guards against this by considering elements in the null distribution
+        that are "close" (within a factor of ``1+1e-14``) to the observed
```

Comment:
factor of `1+1e-14` is not correct anymore

Reply:
Good catch. If we were to commit the suggestions above, we'd have to do this, too.